vnext/ReactUWP/Views/ViewControl.cpp

```diff
@@ -29,70 +22,152 @@ ViewControl::ViewControl()
   return winrt::make_self<ViewControl>();
 }
 
-winrt::Windows::UI::Xaml::Automation::Peers::AutomationPeer ViewControl::OnCreateAutomationPeer()
+winrt::AutomationPeer ViewControl::OnCreateAutomationPeer()
 {
-  auto dyn = winrt::make<winrt::react::uwp::implementation::DynamicAutomationPeer>(*this);
+  auto dyn = winrt::make<DynamicAutomationPeer>(*this);
   return dyn;
 }
 
-} } // namespace react::uwp
+DynamicAutomationPeer::DynamicAutomationPeer(winrt::FrameworkElement const& owner)
```

Comment:
>DynamicAutomationPeer [](start = 23, length = 21)

You may need to keep idl for DynamicAutomationPeer.
There two benefits:
1. implementation signature match with the idl
2. idl will help you generate the winmd file and then be used by C#. (possible for customer to subclass DynamicAutomationPeer?)

Reply:
DynamicAutomationPeer is tied to ViewControl, which is not exposed. There is no reason to expose one and not the other. If and when we decide we need to expose them for subclassing, we need to do it correctly, not by running code-gen in a separate cpp/winrt project and copying the results into this project.

I'm going to be adding a lot to DynamicAutomationPeer for Milestone 2, and I'm not going to use, let alone perpetuate, using the broken process that was used to create it.